### ISSUE TITLE
Unexpected behavior when closing tabs in Finder

### ISSUE THREAD

**Issue by andreaswilli** [lwouis/alt-tab-macos#696]
**Describe the bug**

When using the `w` shortcut to close a tab in Finder the window disappears from the AltTab overlay. However, this only seems to happen if there is another Finder window to the left. I would expect the Finder window to keep being visible in the overlay as long as it has at least one tab.

AltTab version: 6.10.0
macOS version: 11.0.1 (20B29)

**Screenshots / video**

<img width="500px" src="https://user-images.githubusercontent.com/17298270/99389441-5be8df80-28d7-11eb-9443-a99c0cd43ada.gif" />

**Steps to reproduce the bug**

1. Open a Finder window
1. Open a second tab inside the same Finder window
1. Open a second Finder window
1. Press `alt+tab`
1. Make sure the Finder window with two tabs is selected
1. Press `w` (close window) to close a tab in the Finder window

**Comment by lwouis** [comment]
Hi @andreaswilli! Thank you for sharing this issue

I was able to reproduce it on my machine thanks to your clear steps and recording.

The root cause is that our heuristic to detect if windows are tabbed (there is no public API for that) only triggered on interacting with the window. I assumed that to tab a window, un-tab a tab, merge tabs, un-merge tabs, etc, the user always had to interact with the window. I forgot that AltTab itself can send a close signal without user interaction with the window UI.

I added tab detection to the "window closed" event. It seems to fix the issue. 

This fix will be part of the next release 👍
